releases/release-1.31/release-notes/maps/pr-123513-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 123513
+releasenote:
+  text: Moved ConsistentListFromCache feature flag to beta and enable it by default.
```

Comment:
```suggestion
  text: Moved `ConsistentListFromCache` feature flag to beta and enabled it by default.
```

Reply:
@satyampsoni I think this one got missed!